server/plugin/command.go

```diff
@@ -313,10 +324,30 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		if err := p.SubscribeOrg(ctx, githubClient, args.UserId, owner, args.ChannelId, features, flags); err != nil {
 			return err.Error()
 		}
-
-		return fmt.Sprintf("Successfully subscribed to organization %s.", owner)
+		var subOrgMsg = fmt.Sprintf("Successfully subscribed to organization %s.", owner)
+		if isExcludeRepo {
+			var excludeMsg string
+			for _, val := range strings.Split(excludeRepo, ",") {
+				notificationOffRepoOwner, NotificationOffRepo := parseOwnerAndRepo(val, p.getBaseURL())
```

Comment:
Should empty spaces around `val` be allowed?

Reply:
yes, added trim extra spaces from the name

Reply:
Why isn't `TrimSpace` used?

Reply:
have added this .